lib/dependencies/index.ts

```diff
@@ -22,7 +23,16 @@ export async function getDependencies(
   if (!options) {
     options = {};
   }
-  let command = options.command || 'python';
+
+  if (options.python && options.python !== '2' && options.python !== '3') {
+    throw new Error(
+      'The --python property can be used only as --python=2 or --python=3'
+    );
+  }
+
+  const python = options.python ? `python${options.python}` : 'python';
```

Comment:
so the default is unaffected and is Python2 yeah?

Reply:
Yes, if there's something to improve/that I missed, let me know thanks!

Reply:
worth noting (in the docs) the default is 'python' so that's whatever the user has the python command targeting, which these days is probably 3.

Reply:
worth noting (in the docs) the default is 'python' so that's whatever the user has the python command targeting, which these days is probably 3.

Reply:
Took a look at this and all this is not what we think :)

The `command` is what we will in the end use to run `python --version` or `python script.py`, so for a user it can be *anything*

if. user setup their `local` version to `python2` to mean python version 2 then great, what if they did `python2.7`? We need to allow the command to be in a format perhaps `py*` or do not validate.

In fact it is just a basic alias a user can use *anything*, I suggest we drop this change.

Reply:
Instead **improve the docs** so it is clear what `command means` we mean the alias you locally have fo ra python version